src/pages/me/[username].vue

```diff
@@ -1,10 +1,26 @@
 <template>
-  <Layout page="me"> </Layout>
+  <Layout page="me">
+    <div class="container relative mx-auto">
+      <div class="items-center flex flex-wrap">
+        <div class="w-full lg:w-6/12 px-4 ml-auto mr-auto text-center">
+          <div class="py-4">
+            <h1 class="font-semibold text-5xl underline">the resum∈</h1>
```

Comment:
[Optional] Will this: `Résumé` look better & classier?

Reply:
PR updated as per your feedback.
Please review & resolve this conversation.